cmds/netplan/netplan.go

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"gopkg.in/yaml.v2"
+)
+
+type netplanType struct {
+	Network networkType `yaml:"network"`
+}
+
+type networkType struct {
+	Ethernets map[string]interface{} `yaml:"ethernets,omitempty"`
+	Wifis     map[string]interface{} `yaml:"wifis,omitempty"`
+	Version   int                    `yaml:"version"`
+}
+
+const netplanConfig = "/etc/netplan/00-snapd-config.yaml"
+
+func main() {
```

Comment:
Good suggestion. I'd like to leave to a follow-on PR (for which I will create a trello card). This work would create a new pkg ("netplan") and the command would use it. This would also allow the pkg to be called from other places in code, if that turns out to be appropriate, and to easily refactor for unit tests.